src/main/java/org/jabref/gui/EntryTypeViewModel.java

```diff
@@ -144,7 +143,8 @@ protected Optional<BibEntry> call() throws FetcherException {
 
     public void runFetcherWorker() {
         searchSuccesfulProperty.set(false);
-        fetcherWorker.run();
+        searchingProperty().setValue(true);
```

Comment:
Shouldn't that be kept inside the `call` to have a "proper" timing for knowing that a search is going on. - Maybe at some `fetcherWork.setOnRunning` or something method?

If we need it at that that place, a small comment would be nice why we can set it here to "true"

Reply:
It was, but as it would then on the background thread it does not work out of the box, and you would need to wrap it on the FX Thread again.

Reply:
But maybe overriding onRunnning could work that should be executed in the FX Thread. FEel free to adjust that.

Reply:
Did that - and started the worker after its properties are set: [`089291f` (#10416)](https://github.com/JabRef/jabref/pull/10416/commits/089291f560eb7c7cdd536d1d33e508c61e96a6a2)